templates/elasticsearch/template.yaml

```diff
@@ -0,0 +1,912 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: AWS Service Broker - Amazon Elasticsearch
```

Comment:
can you add the template id `(qs-1poknmeof)` to the end of the description

Reply:
Added ID.